src/yuzu/configuration/configure_network.cpp

```diff
@@ -47,29 +48,42 @@ ConfigureService::ConfigureService(QWidget* parent)
     ui->bcat_source->addItem(QStringLiteral("Boxcat"), QStringLiteral("boxcat"));
 #endif
 
+    ui->network_interface->addItem(QStringLiteral("None"));
```

Comment:
```suggestion
    ui->network_interface->addItem(tr("None"));
```

Since this is a user-facing string.